idd/Energy+.idd.in

```diff
@@ -11612,7 +11847,11 @@ Wall:Interzone,
        \note Zone for the inside of the surface
        \type object-list
        \object-list ZoneNames
-  A4 , \field Outside Boundary Condition Object
+  A4 , \field Space Name
+       \note Space the surface is a part of
+       \type object-list
+       \object-list SpaceNames
+  A5 , \field Outside Boundary Condition Object
```

Comment:
I'm surprised this object doesn't have a min fields given that there are coordinates at the bottom. I wonder what happens if you leave out an input?